### PR TITLE
Fix linkedin backend

### PR DESCRIPTION
**Pull request by euri10**
See https://github.com/authlib/loginpass/issues/6
This fixes them, however i don't know if it's the "right" way to proceed or in the spirit of the lib I'm still discovering.
Let me know

### CODE REVIEW

File: loginpass/linkedin.py [hunk 1]
```diff
@@ -23,15 +34,19 @@ class LinkedIn(OAuthBackend):
         'api_base_url': 'https://api.linkedin.com/v1/',
         'access_token_url': 'https://www.linkedin.com/oauth/v2/accessToken',
         'authorize_url': 'https://www.linkedin.com/oauth/v2/authorization',
-        'client_kwargs': {'scope': 'r_basicprofile r_emailaddress'},
+        'client_kwargs': {
+            'scope': 'r_basicprofile r_emailaddress',
+            'token_endpoint_auth_method': 'client_secret_post',
+        },
+        'compliance_fix': lambda s: _linkedin_compliance_fix(s)
```
Comment (lepture) [inline loginpass/linkedin.py]: No need for lambda

**Comment by lepture** [comment]
It seems ok. I didn’t import the module into init file because I can’t verify it myself.

**Comment by euri10** [comment]
just for my understanding, is LinkedIn the "culprit" in not sending a token_type key with the access_token, expires etc ?

**Comment by lepture** [comment]
I merged it manually. Thanks for your contribution.

**Pull request closed without merging**